lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php

```diff
@@ -653,6 +653,8 @@ private function _parseOptions(SimpleXMLElement $options)
     {
         $array = array();
 
+        $booleanOptions = ['unsigned', 'fixed'];
```

Comment:
Since this constant is not used anywhere else, it can be inlined into the `in_array()` call